apps/src/lib/kits/maker/toolkit.js

```diff
@@ -155,7 +159,7 @@ function disconnect() {
  * @returns {Promise}
  */
 function confirmSupportedBrowser() {
-  if (isCodeOrgBrowser() || (isChrome() && gtChrome33())) {
+  if (isCodeOrgBrowser() || (isChrome() && getChromeVersion() >= 90)) {
```

Comment:
Why does this need to be >= 90 now? 

I'm not sure where to put a note for something like this - maybe just here in github so we can find it in the blame?

To be super fancy, maybe in a constant (`MIN_CHROME_VERSION`?) and _that_ constant have a comment on its line pointing to another external doc with rationale?

Reply:
Thanks Jim - I actually thought about using a constant but then forgot to implement! So will add a constant with a comment.